packages/theme/src/theme/_variables.scss

```diff
@@ -17,13 +17,13 @@ $gray:                   lighten($gray-base, 33.5%) !default; // #555
 $gray-light:             lighten($gray-base, 46.7%) !default; // #777
 $gray-lighter:           lighten($gray-base, 85%) !default;   // #bfbfbf
 
-$brand-primary:         $st-tropaz !default;
+$brand-primary:         $patina !default;
```

Comment:
1. Shouldn't we remove the constant **$limeade** (the previous SUCCESS_GREEN) to make sure it is not used anymore?
2. I am wondering where the **$green-talend** is used ?